_topic_maps/_topic_map.yml

```diff
@@ -927,37 +927,47 @@ Topics:
 - Name: Compliance Operator
   Dir: compliance_operator
   Topics:
+  - Name: Compliance Operator overview
+    File: co-overview
   - Name: Compliance Operator release notes
     File: compliance-operator-release-notes
-  - Name: Supported compliance profiles
-    File: compliance-operator-supported-profiles
-  - Name: Installing the Compliance Operator
-    File: compliance-operator-installation
-  - Name: Updating the Compliance Operator
-    File: compliance-operator-updating
-  - Name: Compliance Operator scans
-    File: compliance-scans
-  - Name: Understanding the Compliance Operator
-    File: compliance-operator-understanding
-  - Name: Managing the Compliance Operator
-    File: compliance-operator-manage
-  - Name: Tailoring the Compliance Operator
-    File: compliance-operator-tailor
-  - Name: Retrieving Compliance Operator raw results
-    File: compliance-operator-raw-results
-  - Name: Managing Compliance Operator remediation
-    File: compliance-operator-remediation
-  - Name: Performing advanced Compliance Operator tasks
-    File: compliance-operator-advanced
-  - Name: Troubleshooting the Compliance Operator
-    File: compliance-operator-troubleshooting
-  - Name: Uninstalling the Compliance Operator
-    File: compliance-operator-uninstallation
-  - Name: Using the oc-compliance plugin
-    File: oc-compliance-plug-in-using
-  - Name: Understanding the Custom Resource Definitions
-    File: compliance-operator-crd
-
+  - Name: Compliance Operator concepts
+    Dir: co-concepts
+    Topics:
+    - Name: Understanding the Compliance Operator
+      File: compliance-operator-understanding
+    - Name: Understanding the Custom Resource Definitions
+      File: compliance-operator-crd
```

Comment:
💯 

Glad to see this is presented to readers up front before they get into the actual walk-through.

Reply:
That's another thing we don't have in the docs is a "quick start" walk through. I like the "main workflow" under the diagram in the [upstream docs](https://github.com/ComplianceAsCode/compliance-operator/blob/master/doc/crds.md). We can also work with the branding team to get an official diagram for the docs, too!